e2e/streaming.spec.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env jest */
+/* global element, by */
+
+describe('Streaming Media View', () => {
+	it('is reachable from the home screen', async () => {
+		await expect(element(by.id('screen-homescreen'))).toBeVisible()
+		await expect(
+			element(by.id('homescreen-button-StreamingView')),
+		).toBeVisible()
+		await element(by.id('homescreen-button-StreamingView')).tap()
+		await expect(element(by.id('screen-homescreen'))).toNotBeVisible()
+		await expect(element(by.id('screen-streaming'))).toBeVisible()
+	})
+
+	it('has the Stream List visible by default', async () => {
+		await expect(element(by.id('screen-homescreen'))).toBeVisible()
+		await expect(
+			element(by.id('homescreen-button-StreamingView')),
+		).toBeVisible()
+		await element(by.id('homescreen-button-StreamingView')).tap()
+		await expect(element(by.id('screen-homescreen'))).toNotBeVisible()
+		await expect(element(by.id('screen-streaming'))).toBeVisible()
+
+		await expect(element(by.id('stream-list'))).toBeVisible()
+	})
+})
+
+describe('KSTO Radio View', () => {
+	it('is reachable under the home screen', async () => {
+		await expect(element(by.id('screen-homescreen'))).toBeVisible()
+		await expect(
+			element(by.id('homescreen-button-StreamingView')),
+		).toBeVisible()
+		await element(by.id('homescreen-button-StreamingView')).tap()
+		await expect(element(by.id('screen-homescreen'))).toNotBeVisible()
+		await expect(element(by.id('screen-streaming'))).toBeVisible()
+
+		await element(by.text('KSTO')).tap()
```

Comment:
I think there’s a way to target tab bar items explicitly

Reply:
Ah yes, after ~10 minutes of digging: [`tabBarTestID`](https://reactnavigation.org/docs/en/material-bottom-tab-navigator.html#tabbartestid) looks like the ticket. We will set that in the `navigationOptions` under the screens in such things in addition to the e.g. `tabBarLabel` and `tabBarIcon` things.